src/App.jsx

```diff
@@ -134,6 +135,94 @@ function App() {
                 each year without running out of money for at least 30 years. This calculation 
```

Comment:
⚖️ **Midlevel Developer**

On line 135, you're using `useState` to manage the modal visibility, which is great! But I noticed all the modal content is inline in `App.jsx`. At my last company, we used the "Single Responsibility Principle" and moved all UI chunks into dedicated components for easier unit testing and separation of concerns. I recommend extracting the modal into its own file and adding a few Jest tests to verify open/close behavior. Bonus points for snapshot testing the modal content!